src/main/kotlin/ecommerce/model/ActiveMemberDTO.kt

```diff
@@ -0,0 +1,6 @@
+package ecommerce.model
+
+data class ActiveMemberDTO(
+    val id: Long,
+    val email: String,
```

Comment:
```suggestion
    @field:Email
    val email: String,
```

Just a small suggestion 😄